pkg/lifecycle/kustomize/pre_kustomize.go

```diff
@@ -130,20 +192,37 @@ func (l *Kustomizer) initialKustomizeRun(ctx context.Context, step api.Kustomize
 		return errors.Wrap(err, "write initial kustomized yaml")
 	}
 
-	if err := l.replaceOriginal(step, built); err != nil {
+	if err := l.replaceOriginal(step.Base, built); err != nil {
+		return errors.Wrap(err, "replace original yaml")
+	}
+
+	return nil
+}
+
+func (l *Kustomizer) runProvidedOverlays(ctx context.Context, base, newBase string) error {
```

Comment:
its not immediately clear what base and newbase are in this context

Reply:
is newbase the path to the external base referenced by the `bases` field in the `kustomization.yaml` in the `base` targeted by the `lifecycle.v1.kustomize`? Or is it the built external base?

Reply:
I'll add a comment, possibly change the variable name